src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+            src="images/logo.png"
+            alt="Moyo logo picture"
+            class="img__logo"
+        >
+      </a>
+      <nav class="nav__bar">
+        <ul class="nav__list">
+          <li>
+            <a href="#Apple" class="nav__link is-active">APPLE</a>
```

Comment:
don't write text in uppercase, make it using css

Reply:
Refactored, thank you. The last two(photo and video) left as it was in the layout(uppercase).